src/states/node.rs

```diff
@@ -991,6 +991,10 @@ impl Node {
         self.gen_pfx_info = Some(gen_pfx_info);
         let _ = self.init_parsec()?; // We don't reset the chain on prefix change.
 
+        // Clear any pending candidate
+        // TODO: maybe also disconnect if currently connected.
```

Comment:
Maybe could tackle now rather than keeping it as a TODO?

Reply:
It'd possibly be redundant tho based on crust changes if we arent tracking non-routing peers via peer_mgr PeerState anyways. This was more a point to consider it if we for some reason didnt do the crust integration like so